estudiantes1718.md

```diff
@@ -18,5 +18,8 @@
 <!-- aquí el listado de miembros del grupo 3 -->
 
 
-## Grupo 4. (¿Tema?)
+## Grupo 4. Diversidad cultural
 <!-- aquí el listado de miembros del grupo 4 -->
+
+- [Jaén Huete, María](htto://mjaenh.github.io)
+- [Franco Gómez, Rocío](http://rofrago.github.io)
```

Comment:
Cada uno debéis añadir solamente vuestro propio nombre; @rofrago debería ser quien añada el suyo. Elimina esta línea de tu archivo, y entonces haré merge de tu código en el branch `master`.